modules/account-lib/src/coin/stx/transaction.ts

```diff
@@ -0,0 +1,219 @@
+import BigNumber from 'bignumber.js';
```

Comment:
This also seems to overlap with https://github.com/BitGo/BitGoJS/pull/1055/files - can you pull this out of this PR as well? You can rebase on the other PR if that makes life easier

Reply:
only want to be focused on the contract builder stuff here

Reply:
i created this branch off the feat/stx-builders and this PR is to merge in master, so as long as feat/stx-builders in not merged in master, these changes would appear in this branch.

Reply:
What's the status on that PR? Is there a lot of work remaining to get it merged. I agree with @wesgraham, it would be good to keep the contract builder and keychain changes separated.